doc/api/buffer.markdown

```diff
@@ -1397,7 +1397,8 @@ console.log(`${len} bytes: ${buf.toString('utf8', 0, len)}`);
 
 Writes `value` to the Buffer at the specified `offset` with specified endian
 format (`writeDoubleBE()` writes big endian, `writeDoubleLE()` writes little
-endian). The `value` argument must be a valid 64-bit double.
+endian). The `value` argument *should* be a valid 64-bit double. Behavior is
+unspecified if `value` is anything other than a 64-bit double.
```

Comment:
I think "Behavior is not defined" would be better, since the word "undefined" might confuse basic level english speakers into thinking `undefined` will get written into the buffer somehow. Though TBH "unspecified" looks fine to me.